test/performance/visualization/grafana-dashboard.json

```diff
@@ -0,0 +1,3324 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "datasource",
+          "uid": "grafana"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 5,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "datasource": {
+        "type": "influxdb",
+        "uid": "6BTInGR4k"
+      },
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 15,
+      "targets": [
+        {
+          "datasource": {
+            "type": "influxdb",
+            "uid": "6BTInGR4k"
```

Comment:
When trying to deploy the dashboard I hit this one: https://github.com/grafana/grafana/issues/48611.
We probably need to have a version without the uid.

Reply:
Actually replacing the above `"datasource": "InfluxDB",` the default name of the connection worked for me, check [this](https://github.com/grafana/grafana/discussions/45230#discussioncomment-4088203).

Reply:
I did add it as a variable. Then it should work no matter what/if the grafana even has the datasource or not.